source/NVDAObjects/UIA/__init__.py

```diff
@@ -379,16 +380,16 @@ def _get_bookmark(self):
 		UIAHandler.UIA_TabItemControlTypeId,
 		UIAHandler.UIA_TextControlTypeId,
 		UIAHandler.UIA_SplitButtonControlTypeId
-	} if UIAHandler.isUIAAvailable else None

```

Comment:
This doesn't look like it should be in this change.

Reply:
You're correct, it makes sense to split this out.